usdf/batch.rst

```diff
@@ -201,6 +202,26 @@ For running at S3DF, the following ``site`` specification can be used in the BPS
          condor:
            +Walltime: 7200
 
+allocateNodes auto
+------------------
+
+The ``ctrl_execute`` package now provides an ``allocateNodes --auto`` mode in which the user does not have to specify the number of glideins to run. This mode is not the default, and must be explicitly invoked. In this mode the user's idle jobs in the htcondor queue will be detected and an appropriate number of glideins submitted. At this stage of development the allocateNodes auto is used in conjuction with a bash script that runs alongside a BPS workflow, workflows, or generic HTCondor jobs.  The script will invoke allocateNodes auto at regular intervals to submit the number of glideins needed by the workflow(s) at the particular time.  A sample ``service.sh`` script is::
```

Comment:
The allocateNodes does it once and quits. 
Rather than build the looping in allocateNodes, the simple service.sh is the natural wrapper for 
a SERVICE node in a Dag anyway, so I think it fits.